app/assets/translations/appConstants/bebbo/TJru.ts

```diff
@@ -457,7 +458,7 @@ const TJru = {
                 tncScreenContent: "Беббо не собирает и не хранит никакую личную информацию. Все введенные вами данные сохраняются только на вашем телефоне.",
                 welcomeText:"Добро пожаловать!",
                 letGetStartedText:"ДАВАЙТЕ НАЧНЕМ!",
-                searchInBebboText : "Search in Bebbo",
+                searchInBebboText : "Search",
```

Comment:
You should probably apply the same change in TJtg.ts file.

Reply:
Please ignore 'search' string for now as we are not using it.